backend/benches/impact.rs

```diff
@@ -1,7 +1,25 @@
 use backend::test_fixtures::NeighbourhoodFixture;
 use criterion::{criterion_group, criterion_main, Criterion};
 
-fn benchmark_impact(c: &mut Criterion) {
+fn benchmark_impact_initial_build(c: &mut Criterion) {
+    for fixture in [NeighbourhoodFixture::DUNDEE] {
+        let mut map = fixture.map_model().unwrap();
+        let fast_sample = false;
+
+        c.benchmark_group(fixture.savefile_name)
+            .sample_size(10)
+            .bench_function("predict impact - initial build", |b| {
+                b.iter(|| {
+                    map.rebuild_router(1.0);
```

Comment:
I'm not actually sure if this new bench is useful. TBH because there is some memoized state involved, it's a little tricky. Most of my measurements were done in the browser using the new console.log in Loading.svelte